mifosng-android/src/main/java/com/mifos/objects/client/Client.java

```diff
@@ -110,6 +112,14 @@ public void setGroupId(int groupId) {
         this.groupId = groupId;
     }
 
+    public String getMobileNo() {
+        return mobileNo;
+    }
+
+    public void setMobileNo(String mobileNo) {
+        this.mobileNo = mobileNo;
```

Comment:
If mobile number is not present, provide some default String like 'Not Available'.

Reply:
If the mobile number is not present then mobile no. TableRow will be hidden. ```(TextUtils.isEmpty(client.getMobileNo())) rowMobileNo.setVisibility(GONE);```